src/Plugin.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace TestHelper;
+
+use Cake\Core\BasePlugin;
+use Cake\Routing\RouteBuilder;
+use Cake\Routing\Route\DashedRoute;
+
+/**
+ * Backend Plugin
+ *
+ * @author Flavius
+ * @version 1.0
```

Comment:
Should we keep these tags here?

Reply:
I don't care about the tags, it was copy/pasta from my backend plugin